source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,153 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v6.0.
 
 Latest Mattermost Releases:
+- [Release v7.6 - Feature Release](#release-v7-6-feature-release)
 - [Release v7.5 - Feature Release](#release-v7-5-feature-release)
 - [Release v7.4 - Feature Release](#release-v7-4-feature-release)
 - [Release v7.3 - Feature Release](#release-v7-3-feature-release)
-- [Release v7.2 - Feature Release](#release-v7-2-feature-release)
 - [Release v7.1 - Extended Support Release](#release-v7-1-extended-support-release)
 
+## Release v7.6 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release Day: December 15, 2022**
+
+### Important Upgrade Notes
+
+ - Plugins with a webapp component may need to be updated to work with Mattermost v7.6 release and the updated ``React v17`` dependency. This is to avoid plugins crashing with an error about ``findDOMNode`` being called on an unmounted component. While our `starter template <https://github.com/mattermost/mattermost-plugin-starter-template>`_ depended on an external version of ``React``, it did not do the same for ``ReactDOM``. Plugins need to update their ``webpack.config.js`` directives to externalize ``ReactDOM``. For reference, see https://github.com/mattermost/mattermost-plugin-playbooks/pull/1489. Server-side only plugins are unaffected. This change can be done for existing plugins any time prior to upgrading to Mattermost v7.6 and is backwards compatible with older versions of Mattermost.
+ - Denormalized Threads table by adding the ``TeamId`` column. See details for schema changes https://github.com/mattermost/mattermost-server/pull/20915#issuecomment-1282157291.
```

Comment:
@iomodo Also, has this been tested on community?

Reply:
>Also, has this been tested on community?

This has been deployed on community, so I think the migration has happened and the code is soaking on community. This was introduced as part of Threads Everywhere, so tagging @lieut-data in case there's any additional context.

>Is there a place in the docs to add the content at https://github.com/mattermost/mattermost-server/pull/20915#issuecomment-1282157291?


I think Agniva might still be on PTO, I'm not sure. Someone on the server platform team might know how we surface these docs to customers - @isacikgoz

Reply:
I am back :) @amyblais - these should be surfaced in the upgrade notes as usual. It is part of the schema migration template that every dev should fill out for any migration change.

Reply:
I've added it to the upgrade notes now.